### DISCUSSION TITLE
XQuartz after bash upgrade

### DISCUSSION THREAD

**Discussion by squareturn** [XQuartz/XQuartz#346] [Q&A]
Hi, Can anyone help me troubleshoot a problem.

I have a new M1 MBP.  I installed XQuartz and it was working fine.  I upgraded the stock `bash` 3.2.57(1)-release to 5.2.15(1)-release via homebrew and `chsh -s` to use the updated version.  With the upgraded bash, XQuartz no longer runs.  It appears that `launchd` is restarting it, but there's no information in `~/Library/Logs/X11/org.xquartz.log` indicating what's failing.

I'd appreciate any troubleshooting suggestions.

**Comment by dazuelos** [comment]
Basic suggestion:

1. stop the automatic `launchctl` starting,
2. check where is pointing `sh` symbolic link  in:
```
/private/var/select
```
3. start it manually:
```
/bin/sh -x /opt/X11/bin/startx -- /opt/X11/bin/Xquartz
```
